internal/test/mcp.go

```diff
@@ -12,6 +12,13 @@ import (
 	"golang.org/x/net/context"
 )
 
+func McpInitRequest() mcp.InitializeRequest {
+	initRequest := mcp.InitializeRequest{}
+	initRequest.Params.ProtocolVersion = mcp.LATEST_PROTOCOL_VERSION
```

Comment:
just curiois: new sdk requires `2025-06-18` ?

Reply:
No, that was already there:

https://github.com/containers/kubernetes-mcp-server/blob/6342379505a0726821c5d1420102a8969e99aadb/internal/test/mcp.go#L31-L33

Just moved it around and made it reusable.